cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/s3/S3StorageClient.java

```diff
@@ -55,4 +56,11 @@ public S3Client getNativeClient() {
   public StorageType.Type getStorageType() {
     return S3_TYPE;
   }
+
+  @Override
+  public boolean pathExists(String path) {
```

Comment:
What is the intended usage of this existence check?
1. Do you want to check whether the bucket exists or not?
2. Or, do you want to check whether any blob for the table has been created yet or not?

The two checks above are very different. For 1 you should check existence of bucket. For 2 you should check existence of objects with a prefix that way you are doing it now.

I would imagine that you intend to check that the bucket or root location has been created and hence 1 and not 2. Is that right?

Reply:
The intention is to whether directory exists so it is check 2.
Current code passes the absolute path until table 
-> blobfs://bucket_name/data/openhouse/db/t-uuid (blobfs)
-> s3://bucket_name/data/openhouse/db/t-uuid (s3)
-> /data/openhouse/db/t-uuid (hdfs)

Reply:
This is no longer valid. See https://github.com/linkedin/openhouse/pull/256#issuecomment-2492717671